internal/gengapic/client_init_test.go

```diff
@@ -382,13 +382,13 @@ func TestClientInit(t *testing.T) {
 			serv:      servLRO,
 			parameter: proto.String("go-gapic-package=path;mypackage"),
 			imports: map[pbinfo.ImportSpec]bool{
-				{Path: "context"}:                                                                  true,
-				{Path: "google.golang.org/grpc"}:                                                   true,
-				{Path: "google.golang.org/grpc/metadata"}:                                          true,
 				{Name: "gtransport", Path: "google.golang.org/api/transport/grpc"}:                 true,
 				{Name: "longrunningpb", Path: "google.golang.org/genproto/googleapis/longrunning"}: true,
 				{Name: "lroauto", Path: "cloud.google.com/go/longrunning/autogen"}:                 true,
 				{Name: "mypackagepb", Path: "github.com/googleapis/mypackage"}:                     true,
+				{Path: "context"}:                         true,
+				{Path: "google.golang.org/grpc"}:          true,
+				{Path: "google.golang.org/grpc/metadata"}: true,
```

Comment:
can you run `gofmt -w .` I think these should be aligned with the other values...not sure why they got moved down here from up there in the first place? Maybe revert this bit

Reply:
I'm not sure either. `gofmt` does not want to change the indentation, and it reverts the change when I attempt to fix it up manually.
I don't think I can revert this bit; I needed to remove the dependency on `strings` for these cases, and apparently that triggered the gofmt failure.

Reply:
huh ok I wonder if the lines are too long and it doesn't try to align anymore. All good!